docs/src/gui/qtvcp-vcp-panels.adoc

```diff
@@ -244,22 +257,9 @@ net s32-in1        test_panel.doublescale_1-s   classicladder.0.s32in-00
 start
 ----
 
-In this case we load qtvcp using -Wn; which waits for the panel to
-finish loading before continuing to run the next HAL command. This is so
-the HAL pins from the panel are finished in case the are used in the
-rest of the file.
-
-
-== More Information
-
-<<cha:qtvcp-widgets,QtVCP Widgets>>
-
-<<cha:qtvcp-libraries,QtVCP Libraries>>
-
-<<cha:qtvcp-code,QtVCP Handler File Code Snippets>>
-
-<<cha:qtvcp-development,QtVCP Development>>
-
-<<cha:qtvcp-custom-widgets,QtVCP Custom Designer Widgets>>
+<1> In this case we load `qtvcp` using *`-Wn`* which waits for the panel
```

Comment:
Is `<1>` needed in the code? 
And I guess "1."  should be a footnote referring to "<1>" ?
If I am right, it is more guessed and not really clear.

Reply:
See my other answer about callouts